README.md

```diff
@@ -55,6 +67,11 @@ Please refer to the respective terms of service and copyrights.
 ## Options
 
 - `Topic` is the topic of the GPS measurements.
+- `Map Transform Type` selects between the `Map frame` mode and `UTM frame` mode (see section `Usage`).
+- `Map Frame` is the map frame rigidly attached to the world with ENU convention. 
+- `UTM Frame` is the frame that represents UTM coordinate frame.
+- `UTM Zone` is the zone used by the `UTM frame`. Value `-1` triggers autodetection of zone and this property is then
```

Comment:
One reason we sometimes fallback to lat/lon is difficulties with utm zones.

What happens when the robot starts up, crosses an UTM zone and only then you open up rviz.

Reply:
That's why the UTM zone is configurable. If your rviz has wrong zone, just fix it in the display config.

I haven't yet found a good standardized way of specifying the UTM zone and hemisphere used by the current `utm` frame (e.g. the one published from robot_localization). I think the best is still subscribing to fix messages and determining the zone from them. But it's true that this case would fail if you started rviz after a zone boundary has been crossed... Maybe there could be some cross-check via TF to verify that the computed UTM position of the GNSS receiever and the lat/lon-derived UTM position are roughly the same (+- 100 km :) ).

Reply:
The `earth` frame from REP-105 would resolve this, but I haven't actually seen it used anywhere...

Reply:
I've been meaning to extend `robot_localization` with the option to output multiple utm frames (with a zone number).
This would make this problem also easier: https://github.com/cra-ros-pkg/robot_localization/issues/698#issuecomment-1056550816

Ahwel, it's a good start I think.

Reply:
I've seen that comment and it makes sense to me.